changelog.d/pr-7094.md

```diff
@@ -0,0 +1,6 @@
+### Bug Fixes
+
+- Stop rendering the output of internal `subdatset()` call in the
+  results of `run_procedure()`.
+  Fixes https://github.com/datalad/datalad/issues/7091 via
+  https://github.com/datalad/datalad/pull/7094 (by @mslw & @mih)
```

Comment:
might want to make `@mslw` and `@mih` into hyperlinks

Reply:
Done. Also changed issue/PR into formatted links. I started with an old snippet template, that's why I didn't do it right away.